trl/trainer/judges.py

```diff
@@ -303,3 +343,27 @@ def get_rank(prompt, candidates):
 
         # Return the ranks
         return ranks
+
+
+class MixtureOfConstraintJudges(BaseConstraintJudge):
```

Comment:
Same question here: `MixtureConjunctionJudge`? `ConjunctionJudge`? `AllTrueJudge`?

Reply:
`AllTrueJudge` seems cool in my opinion